<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogStation</class>
 <widget class="QDialog" name="DialogStation">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WatchRR -- Station</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelStation">
     <property name="text">
      <string>Station:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="editStation"/>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="labelStationInfo">
     <property name="text">
      <string notr="true">Blindern (12345m)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelFrom">
     <property name="text">
      <string>From:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="MiDateTimeEdit" name="dateFrom">
     <property name="displayFormat">
      <string notr="true">yyyy-MM-dd hh:mm</string>
     </property>
     <property name="timeSpec">
      <enum>Qt::UTC</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelTo">
     <property name="text">
      <string>To:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="MiDateTimeEdit" name="dateTo">
     <property name="displayFormat">
      <string notr="true">yyyy-MM-dd hh:mm</string>
     </property>
     <property name="timeSpec">
      <enum>Qt::UTC</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelType">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOK">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="comboType"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MiDateTimeEdit</class>
   <extends>QDateEdit</extends>
   <header>MiDateTimeEdit.hh</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogStation</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOK</sender>
   <signal>clicked()</signal>
   <receiver>DialogStation</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editStation</sender>
   <signal>editingFinished()</signal>
   <receiver>DialogStation</receiver>
   <slot>onEditStation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateFrom</sender>
   <signal>editingFinished()</signal>
   <receiver>DialogStation</receiver>
   <slot>onEditTime()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateTo</sender>
   <signal>editingFinished()</signal>
   <receiver>DialogStation</receiver>
   <slot>onEditTime()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>DialogStation</receiver>
   <slot>onSelectType(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onEditStation()</slot>
  <slot>onEditTime()</slot>
  <slot>onSelectType(int)</slot>
 </slots>
</ui>
